app/models/mixins/authentication_mixin.rb

```diff
@@ -273,6 +273,27 @@ def authentication_check(*args)
     return status == :valid, details
   end
 
+  def authentication_check_queue(*args)
```

Comment:
Good point @AparnaKarve.

The concept was that classes could dictate what authentication types it used to authenticate via `authentications_to_validate`.

Another way is to pass the types, see [here](https://github.com/ManageIQ/manageiq/blob/4bdabf303ac33dbc95466c49a7a66a081494561c/spec/models/mixins/authentication_mixin_spec.rb#L329).  If you don't specify the types, then use `authentications_to_validate` will be used.